spec/stashcache.spec

```diff
@@ -53,26 +49,31 @@ Group: Grid
 Summary: Metapackage for a cache server
 
 Requires: xrootd-server >= 1:4.6.1
-Requires: xrootd-lcmaps >= 1:1.3.3
+Requires: xrootd-lcmaps >= 1.3.3
 Requires: %{name}-daemon
 
 %description cache-server
 %{summary}
 
 %prep
 %setup -q
+%if 0%{?el6}
+echo "*** This version does not build on EL 6 ***"
+exit 1
+%endif
 
 %install
 mkdir -p %{buildroot}%{_sysconfdir}/xrootd
 make install DESTDIR=%{buildroot}
-for src in "./configs/xrootd-stashcache-origin-server.cfg.in" \ 
-           "./configs/xrootd-stashcache-cache-server.cfg.in"; do
+for src in "./configs/xrootd-stashcache-origin-server.cfg.in" "./configs/xrootd-stashcache-cache-server.cfg.in"; do
     dst=$(basename "$src" .cfg.in)
     sed -i -e "s#@LIBDIR@#%{_libdir}#" "$src"
     sed -e "s#@REDIRECTOR@#%{redirector_prod}#" \
         "$src" > "%{buildroot}%{_sysconfdir}/xrootd/${dst}.cfg"
+    chown xrootd:xrootd %{buildroot}%{_sysconfdir}/xrootd/${dst}.cfg
```

Comment:
Instead, would this do a job simply add line per origin-server and cache-server `%files` section like:
`%defattr(-,xrootd,xrootd)`